### ISSUE TITLE
Expose validation result in `onSelect`

### ISSUE THREAD

**Issue by KCraw** [dubert/react-kronos#44]
When controlling visibility, there is currently no way to know whether validation succeeded or failed when `props.onSelect` is called.

The internal `onSelect` uses toggles state.visible; however, if you have enabled `controlVisibility`, this will be out of sync with the actual visibility. 

The `props.onSelect` callback should set the appropriate visibility, but it would be nice to know whether the validation succeeded or not.

**Comment by felixakiragreen** [comment]
Would you like for this to be a parameter in the callback? For example change: https://github.com/dubert/react-kronos/blob/master/src/index.js#L342

to:

``` js
if (this.props.onSelect) this.props.onSelect(datetime, willBeVisible, didValidate)
```

(and obviously define the didValidate variable)

**Comment by KCraw** [comment]
Yeah, that would be great. That way we can follow similar logic to the
default of not closing the input if it was somehow invalid. Thanks!

**Comment by felixakiragreen** [comment]
_Just released **1.6.0**!_

`onSelect` will now include `(datetime, visible, shouldClose)`. `shouldClose` will inform you whether it validated.
